app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -84,6 +105,11 @@ public void insertShape(MapPoint point) {
         database.insert(DBConstants.DB_TABLE_SHAPE, null, cv);
     }
 
+
+    /**
+     * Inserts QR data to QRs table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.